README.md

```diff
@@ -1,14 +1,50 @@
-This is a [Giter8][g8] template for Akka in Java.
+# Akka Quickstart with Java
 
+## Quickstart guide
+
+This example is described in the [Akka Quickstart with Java guide](http://developer.lightbend.com/guides/akka-quickstart-java/)
+
+The Hello World example illustrates Akka basics. Within 30 minutes, you should be able to download and run the example and use this guide to understand how the example is constructed. This will get your feet wet, and hopefully inspire you to dive deeper into the wonderful sea of Akka!
+
+## Giter8 template
+
+It can also be used as a [Giter8][g8] template for Akka with Java.
+
+Prerequisites:
+- JDK 8
+- [sbt][sbt] 0.13.13 or higher ([download here][sbt_download])
+
+Open a console and run the following command to apply this template:
+ ```
+sbt -Dsbt.version=0.13.15 new https://github.com/akka/akka-java-seed.g8
+ ```
+
+This template will prompt for the following parameters. Press `Enter` if the default values suit you:
+- `name`: Becomes the name of the project.
+
+Once inside the project folder, to run Hello World with:
 ```
-sbt new akka/akka-java-seed.g8
+sbt run
 ```
 
-License
--------
-Written in 2014-2016 by Lightbend, Inc.
-To the extent possible under law, the author(s) have dedicated all copyright and related and neighboring rights to
-this template to the public domain worldwide. This template is distributed without any warranty.
-See <http://creativecommons.org/publicdomain/zero/1.0/>.
+This template also provides build descriptors for maven and gradle. You can use any of the following commands to run 
+the application:
+```
+mvn exec:exec
```

Comment:
We need to explicitly call `compile` before trying to run. So full command would be:

    mvn compile exec:exec